packages/backend/src/tests/app.ts

```diff
@@ -76,10 +77,20 @@ export const createTestApp = async (
       })
     }
   })
-  const authLink = setContext((_, { headers }) => {
+  const authLink = setContext((request, { headers }) => {
+    const requestBody = {
+      variables: request.variables,
+      operationName: request.operationName,
+      query: print(request.query)
+    }
     return {
       headers: {
-        ...headers
+        ...headers,
+        signature: generateApiSignature(
+          config.adminApiSecret,
+          config.adminApiSignatureVersion,
+          requestBody
+        )
```

Comment:
edit: because of our replay attack [checks](https://github.com/interledger/rafiki/pull/3177#issuecomment-2541724587), going to use `if (config.env === "test") don't add the auth middleware` method